RollbarNotifier/Sources/RollbarNotifier/RollbarPayloadRepository.m

```diff
@@ -15,11 +15,11 @@
 
 static int defaultOnSelectCallback(void *info, int columns, char **data, char **column)
 {
-    RollbarSdkLog(@"Columns: %d", columns);
-    for (int i = 0; i < columns; i++) {
-        RollbarSdkLog(@"Column: %s", column[i]);
-        RollbarSdkLog(@"Data: %s", data[i]);
-    }
+//    RollbarSdkLog(@"Columns: %d", columns);
```

Comment:
I'd like for our users to decide whether Rollbar spits output to the console or not. Right now, it spits everything and makes the console unusable. I commented out these because I want to restore them later behind some on/off flag.